tools/gen.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+qt \
+--in=$(dirname $PWD)/google-cloudevents/jsonschema \
+--out=$PWD \
```

Comment:
So does qt automatically add the `src/main` part, or should this be `$PWD/src/main`?

Should the script delete existing source files first perhaps? If a type is removed (before it's declared stable) it would be good to avoid accidentally leaving it lying around. (We've only just fixed this for C# API generation...)

Reply:
The whole directory path is now moved to a new folder which is expected by this repo.

I've changed the `gen.sh` script to be completely isolated, so you should be able to run this multiple times without changes.

- The script deletes existing source files, then gens the source code
- The script moves generated code to the correct directory, fixes the package